client-portal/modules/card/components/Detail.tsx

```
@@ -302,38 +302,38 @@
             </Card>
           </div>
 
-          <div className="col-md-7">
-            <ControlLabel>Description</ControlLabel>
+          <div className='col-md-7'>
+            <ControlLabel>{__('Description')}</ControlLabel>
             <Card>
               {description ? (
                 <Description
                   dangerouslySetInnerHTML={{ __html: description }}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)